seaworthy/tests/test_utils.py

```diff
@@ -0,0 +1,9 @@
+from testtools.assertions import assert_that
+from testtools.matchers import Equals
+
+from ..utils import resource_name
```

Comment:
The `flake8-import-order` thing was complaining with an absolute import for some reason and I was too lazy to figure out why :/